azure-pipelines-integration-scouting.yml

```diff
@@ -0,0 +1,14 @@
+# Run branch daily at midnight UTC
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily midnight build
+  branches:
+    include:
+    - master-vs-deps
```

Comment:
do we want to include other branches?  I could imagine scenarios where things break in master but not master-vs-deps, but maybe that is too complicated

Reply:
Happy to change this to master if people prefer. We can expand the list as much as we think is necessary.

Reply:
Moved this configuration to the pipeline itself. The scouting pipeline reuses the existing integration yaml but overrides the triggers.